how-to/add-mods/curseforge-app.md

```diff
@@ -0,0 +1,31 @@
+---
+hidden: true
+---
+
+# CurseForge App
+
+{% tabs %}
+{% tab title="Through the CurseForge App" %}
+1. Open the **CurseForge App**
+2. Select **Minecraft** from the grid or from the sidebar
+3. In **My Modpacks**, click on **Fabulously Optimized**
+4. Click on the **three dots** button
+5. Select **Profile Options**
```

Comment:
```suggestion
5. Select **⋮☰ Profile Options**
```
I'd like to keep (or even add) the symbols/emoji where possible, because that helps people find the button.